WordPress/Classes/Models/Blog/Blog+Editor.swift

```diff
@@ -45,4 +46,39 @@ extension Blog {
     @objc public var isGutenbergEnabled: Bool {
         return editor == .gutenberg
     }
+
+    /// - warning: Decoding can take a non-trivial amount of time.
+    func getBlockEditorSettings() -> [String: Any]? {
+        guard let data = rawBlockEditorSettings?.data else {
```

Comment:
The changes ensures that when `Blog` entity is loaded, the settings blob it not loaded in memory. It is achieved by using a relationship (see `BlobEntity`). The entity is also configured to store large blobs in external storage, which is more efficient than putting it in a database.

In the previous implementation, the blob was stored in the `options` dictionary, which is designed for tiny key-value entities and can no be used for large blobs for performance reasons.

Reply:
The benefits of the new blob store make a lot of sense.